media-source-respec.html

```diff
@@ -1413,10 +1487,18 @@ <h4><dfn>Initialization Segment Received</dfn></h4>
               <ol>
                 <li>If the [=initialization segment=] contains tracks with codecs the user agent does not support, then run the [=append error=] algorithm and abort these steps.
                   <p class="note">User agents MAY consider codecs, that would otherwise be supported, as "not supported" here if the codecs were not
-                    specified in the <var>type</var> parameter passed to {{MediaSource/addSourceBuffer()}}. <br>
+                    specified in <var>type</var> parameter passed to
+                    (a) the most recently successful {{SourceBuffer/changeType()}} on this {{SourceBuffer}} object, or
+                    (b) if no successful {{SourceBuffer/changeType()}} has yet occurred on this object, the {{MediaSource/addSourceBuffer()}}
+                    that created this {{SourceBuffer}} object.
                     For example, MediaSource.isTypeSupported('video/webm;codecs="vp8,vorbis"') may return true, but if
```

Comment:
I see that this PR does not touch this part, but I would also wrap `MediaSource.isTypeSupported('video/webm;codecs="vp8,vorbis"')` and following parameter mentions in `<code>`.

Reply:
Good catches. Thank you. This one is fixed by ff03151